server/api/db/sqldb/models.py

```diff
@@ -769,6 +771,61 @@ class TimeWindowTracker(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.key}"
 
+    class ModelEndpoint(Base, mlrun.utils.db.HasStruct):
+        __tablename__ = "model_endpoints"
+        __table_args__ = (
+            UniqueConstraint(
+                "project", "name", "uid", name="_mep_uc_2"
+            ),  # create time? in order to save old MEP
```

Comment:
I guess you may have forgotten to remove this comment. Since the `uid` is generated through the hash process (see `fill_model_endpoint_hash`), which includes the creation timestamp, it's redundant, right?